src/Server.js

```diff
@@ -71,10 +75,28 @@ function middleware(bundler) {
       }
     }
 
-    function send500() {
-      res.setHeader('Content-Type', 'text/plain; charset=utf-8');
+    function send500(error) {
+      res.setHeader('Content-Type', 'text/html; charset=utf-8');
       res.writeHead(500);
-      res.end('🚨 Build error, check the console for details.');
+      let errorMesssge = '<h1>🚨 Build Error</h1>';
+      if (process.env.NODE_ENV === 'production') {
+        errorMesssge += '<p><b>Check the console for details.</b></p>';
+      } else {
+        const {message, stack} = prettyError(error, {color: true});
+        errorMesssge += `<p><b>${message}</b></p>`;
+        if (stack) {
+          errorMesssge += `<div style="background: black; padding: 1rem;">${ansiToHtml.toHtml(
+            stack
+          )}</div>`;
+        }
+      }
+      res.end(
```

Comment:
Why not put use one template string here instead of an array?

Reply:
This is to make sure we're not sending extra whitespaces or not ending up with awkward code indentations. Depending on project coding style I can update.

Reply:
Ah, alright, I don't think the weird indentation matters, but that's personal taste.